src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public enum Colors {
```

Comment:
```suggestion
public enum Color {
```

enum it is a class, and we use singular for class names